### ISSUE TITLE
[General] Unable to build newserv

### ISSUE THREAD

**Issue by Matt-Swift** [fuzziqersoftware/newserv#296]
This is on Debian GNU/Linux 12.4.

```
In file included from /root/newserv/src/LevelTable.cc:1:
/root/newserv/src/LevelTable.hh:102:33: error: field ‘base_stats’ has incomplete type ‘std::array<CharacterStats, 9>’
  102 |   std::array<CharacterStats, 9> base_stats;
      |                                 ^~~~~~~~~~
In file included from /usr/include/c++/12/bits/unique_ptr.h:36,
                 from /usr/include/c++/12/memory:76,
                 from /root/newserv/src/LevelTable.hh:5:
/usr/include/c++/12/tuple:1595:45: note: declaration of ‘struct std::array<CharacterStats, 9>’
 1595 |   template<typename _Tp, size_t _Nm> struct array;
      |                                             ^~~~~
/root/newserv/src/LevelTable.hh:103:32: error: field ‘level_100_stats’ has incomplete type ‘std::array<LevelTableV2::Level100Entry, 9>’
  103 |   std::array<Level100Entry, 9> level_100_stats;
      |                                ^~~~~~~~~~~~~~~
/usr/include/c++/12/tuple:1595:45: note: declaration of ‘struct std::array<LevelTableV2::Level100Entry, 9>’
 1595 |   template<typename _Tp, size_t _Nm> struct array;
      |                                             ^~~~~
/root/newserv/src/LevelTable.hh:104:30: error: field ‘max_stats’ has incomplete type ‘std::array<PlayerStats, 9>’
  104 |   std::array<PlayerStats, 9> max_stats;
      |                              ^~~~~~~~~
/usr/include/c++/12/tuple:1595:45: note: declaration of ‘struct std::array<PlayerStats, 9>’
 1595 |   template<typename _Tp, size_t _Nm> struct array;
      |                                             ^~~~~
/root/newserv/src/LevelTable.hh:105:51: error: field ‘level_deltas’ has incomplete type ‘std::array<std::array<LevelStatsDelta, 200>, 9>’
  105 |   std::array<std::array<LevelStatsDelta, 200>, 9> level_deltas;
      |                                                   ^~~~~~~~~~~~
/usr/include/c++/12/tuple:1595:45: note: declaration of ‘struct std::array<std::array<LevelStatsDelta, 200>, 9>’
 1595 |   template<typename _Tp, size_t _Nm> struct array;
      |                                             ^~~~~
/root/newserv/src/LevelTable.hh:117:52: error: field ‘level_deltas’ has incomplete type ‘std::array<std::array<LevelStatsDelta, 200>, 12>’
  117 |   std::array<std::array<LevelStatsDelta, 200>, 12> level_deltas;
      |                                                    ^~~~~~~~~~~~
/usr/include/c++/12/tuple:1595:45: note: declaration of ‘struct std::array<std::array<LevelStatsDelta, 200>, 12>’
 1595 |   template<typename _Tp, size_t _Nm> struct array;
      |                                             ^~~~~
/root/newserv/src/LevelTable.hh:129:34: error: field ‘base_stats’ has incomplete type ‘std::array<CharacterStats, 12>’
  129 |   std::array<CharacterStats, 12> base_stats;
      |                                  ^~~~~~~~~~
/usr/include/c++/12/tuple:1595:45: note: declaration of ‘struct std::array<CharacterStats, 12>’
 1595 |   template<typename _Tp, size_t _Nm> struct array;
      |                                             ^~~~~
/root/newserv/src/LevelTable.hh:130:52: error: field ‘level_deltas’ has incomplete type ‘std::array<std::array<LevelStatsDelta, 200>, 12>’
  130 |   std::array<std::array<LevelStatsDelta, 200>, 12> level_deltas;
      |                                                    ^~~~~~~~~~~~
/usr/include/c++/12/tuple:1595:45: note: declaration of ‘struct std::array<std::array<LevelStatsDelta, 200>, 12>’
 1595 |   template<typename _Tp, size_t _Nm> struct array;
      |                                             ^~~~~
make[2]: *** [CMakeFiles/newserv.dir/build.make:601: CMakeFiles/newserv.dir/src/LevelTable.cc.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:110: CMakeFiles/newserv.dir/all] Error 2
make: *** [Makefile:146: all] Error 2
```

**Comment by fuzziqersoftware** [comment]
I fixed this just now (bf26e437). Is it still broken after that commit?

**Comment by Matt-Swift** [comment]
It sure isn't!
